web/src/components/tabs/Tabs/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+
+export default createUseStyles((theme: typeof tokens) => ({
+  wrapper: {
+    display: 'flex',
+    height: ({ size }) => (size === 'small' ? theme.size3X : theme.size4X),
+    justifyContent: ({ align }) => {
+      switch (align) {
+        case 'left':
+          return 'flex'
+        case 'right':
+          return 'flex-end'
+        default:
+          return 'flex'
+      }
+    },
+    position: 'relative',
+    '&::after': {
+      content: '""',
+      position: 'absolute',
+      bottom: 0,
+      left: 8,
+      width: 'calc(100% - 16px)',
+      height: theme.borderWidthTab,
+      backgroundColor: theme.colorBorderTabDivider,
+    },
```

Comment:
This is just because of the bottom line right?

This can be replaced by adding a box shadow to the `wrapper`

`box-shadow: inset 0px -2px 0px rgb(241, 241, 241);`

This will also resolve the zIndex "issue"

The first child should not have padding on the left to keep styles aligned

Reply:
If the first and last child will not have padding on the left and right the outline on focus will look weird. Is it ok @pseudonymcp?

Reply:
<img width="346" alt="Screenshot 2020-11-04 at 11 54 25" src="https://user-images.githubusercontent.com/1459899/98102953-904eab80-1e94-11eb-8514-50fb38415e9f.png">

will look, like this